memongo.go

```diff
@@ -136,6 +140,31 @@ func StartWithOptions(opts *Options) (*Server, error) {
 
 	logger.Debugf("mongod started up and reported a port number after %s", time.Since(startupTime).String())
 
+	// ---------- START OF REPLICA CODE ----------
+	// TODO: see benweissman's comments on the original PR:
+	// https://github.com/benweissmann/memongo/pull/6/files#r566993296
+	if opts.ShouldUseReplica {
+		// TODO: we should extract the mongo binary from the downloaded archive
+		// and use that here.
+		// otherwise the user will get a 127 code error if they don't have it installed
+		mongoCommand := fmt.Sprintf("mongo --port %d --retryWrites --eval \"rs.initiate()\"", opts.Port)
```

Comment:
Can't you just use the mongo binary you downloaded?

Reply:
Yeah, I was able to do that. Are there any incompatibilities that could arise from a version mismatch if one were to do that though? I know next to nothing about the mongo client.

Reply:
there are, but we can assume all mongo in a cluster share the same version, otherwise you have a very bad config